cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
+   median is less sensitive to outliers, although the results are also
+   slightly less smooth and the fact that images are in the range of 0
+   to 1 means that outliers typically will not dominate too strongly
+   anyway.
+-  *%(SM_TO_AVERAGE)s:* A less commonly used option is to completely
+   smooth the entire image, which will create a flat, smooth image where
+   every pixel of the image is the average of what the illumination
+   function would otherwise have been.
+-  *%(SM_SPLINES)s:* This method (*Lindblad and Bengtsson, 2001*) fits
+   a grid of cubic splines to the background while excluding foreground
+   pixels from the calculation. It operates iteratively, classifying
+   pixels as background, computing a best fit spline to this background
+   and then reclassifying pixels as background until the spline
+   converges on its final value.
+-  *%(SM_CONVEX_HULL)s:* This method algorithm proceeds as follows:
+    
+   -  Choose 256 evenly-spaced intensity levels between the minimum and
+      maximum intensity for the image
+   -  Set the intensity of the output image to the minimum intensity of
+      the input image
+   -  Iterate over the intensity levels, from lowest to highest
+    
+      -  For a given intensity, find all pixels with equal or higher
+         intensities
+      -  Find the convex hull that encloses those pixels
+      -  Set the intensity of the output image within the convex hull to
+         the current intensity
+    
+   The Convex Hull method can be used on an image whose objects are
+   darker than their background and whose illumination intensity
+   decreases monotonically from the brightest point.
+    
+**References**
+    
+-  J Lindblad and E Bengtsson (2001) “A comparison of methods for
+   estimation of intensity nonuniformities in 2D and 3D microscope
+   images of fluorescence stained cells.”, Proceedings of the 12th
+   Scandinavian Conference on Image Analysis (SCIA), pp. 264-271
+""" % globals())
 
         self.automatic_object_width = cps.Choice(
-                "Method to calculate smoothing filter size",
-                [FI_AUTOMATIC, FI_OBJECT_SIZE, FI_MANUALLY], doc='''
-            <i>(Used only if a smoothing method other than Fit Polynomial is selected)</i><br>
-            Calculate the smoothing filter size. There are three options:
-            <ul>
-            <li><i>%(FI_AUTOMATIC)s:</i> The size is computed as 1/40 the size of the image or
-            30 pixels, whichever is smaller.</li>
-            <li><i>%(FI_OBJECT_SIZE)s:</i> The size is obtained relative to the width
-            of artifacts to be smoothed.</li>
-            <li><i>%(FI_MANUALLY)s:</i> Use a manually entered value.</li>
-            </ul>''' % globals())
+            "Method to calculate smoothing filter size",
+            [FI_AUTOMATIC, FI_OBJECT_SIZE, FI_MANUALLY], doc="""\               
+*(Used only if a smoothing method other than Fit Polynomial is selected)*
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.